lib/api/include/irods/switch_user.h

```diff
@@ -19,35 +21,33 @@ extern "C" {
 /// - All API operations invoked via the \p RcComm will be carried out as the user that was just switched to.
 /// - Server-to-server connections will be updated or disconnected based on the remote server's version.
 ///
-/// On failure, the RcComm will remain as if this API was not invoked. If interacting with the server appears unstable,
-/// the client should disconnect and establish a new connection. To avoid unknown issues, clients are encouraged to
-/// use a new connection following an error. This provides the highest protection of data.
-///
-/// \param[in] _comm     A pointer to a RcComm.
-/// \param[in] _username \parblock
-/// The name of the iRODS user to become.
-///
-/// The passed string must meet the following requirements:
-/// - The length must not exceed sizeof(UserInfo::userName) - 1 bytes
-/// - It must be null-terminated
-/// - It must be non-empty
-/// - It must not include the zone.
-/// \endparblock
-/// \param[in] _zone     \parblock
-/// The zone of the iRODS user to become.
-///
-/// The passed string must meet the following requirements:
-/// - The length must not exceed sizeof(UserInfo::rodsZone) - 1 bytes
-/// - It must be null-terminated
-/// - It must be non-empty
+/// On failure, the RcComm should be closed to avoid potential issues.
+///
+/// \param[in] _comm  A pointer to a RcComm.
+/// \param[in] _input \parblock A pointer to a SwitchUserInput.
+///
+/// SwitchUserInput member variables:
+/// - \p username: The part of the iRODS username preceding the pound sign. The following requirements must be satified:
+///     - The length must not exceed sizeof(UserInfo::userName) - 1 bytes
+///     - It must be null-terminated
+///     - It must be non-empty
+///     - It must not include the zone
```

Comment:
does this really mean 'it must not include a pound (#) sign'?

Reply:
That is correct.

Reply:
Updated.

See switch_user_types.h for new words.